_sass/_code-styles.sass

```
@@ -10,32 +10,58 @@ code
 
 .highlighter-rouge
 	position: relative
-	button.expand-code-btn
-		position: absolute
-		bottom: -33px
-		right: 0
-		left: 0
-		margin-right: auto
-		margin-left: auto
-		display: flex
-		flex-direction: column
-		justify-content: center
-		align-items: center
-		color: #2a7dec
-		width: 130px
-		height: 30px
-		outline: none
-		div
-			width: 0
-			height: 0
-		div.collapsed
-			border-left: 20px solid transparent
-			border-right: 20px solid transparent
-			border-top: 13px solid #2a7dec
-		div.expanded
-			border-left: 20px solid transparent
-			border-right: 20px solid transparent
-			border-bottom: 13px solid #2a7dec
+	&[class*="expandable"]
+		pre.highlight
+			margin-bottom: 45px
+			&.expanded
+				height: max-content !important
+		button.expand-code-btn
+			position: absolute
+			bottom: -33px
+			right: 0
+			left: 0
+			margin-right: auto
+			margin-left: auto
+			display: flex
+			flex-direction: column
+			justify-content: center
+			align-items: center
+			color: #2a7dec
+			width: 130px
+			height: 30px
+			outline: none
+
+			div.arrow
+				width: 0
+				height: 0
+				border-left: 20px solid transparent
+				border-right: 20px solid transparent
+				border-top: 13px solid #2a7dec
+
+			p.collapse
+				margin-top: 0 !important
```

You can add general styles that hide the content in one place
```suggestion
			.collapse
				margin-top: 0 !important
			.collapse, .expand
			 	display: none
```